Water Drink Water/src/clients/blazor.wa.tbd/Pages/Register.razor

```diff
@@ -0,0 +1,47 @@
+﻿@page "/Register"
+<h3>Register</h3>
+
+<EditForm Model="formData" OnValidSubmit="ExecuteRegistration">
+    <DataAnnotationsValidator/>
+    <div class="container">
+        <div class="row">
+            <div class="col-md-12">
+                <label for="name" class="form-label">Name:</label>
+                <InputText id="name" class="form-control" @bind-Value="formData.Name"/>
+            </div>
+        </div>
+        <div class="row">
+            <div class="col-md-12">
+                <label for="email" class="form-label">Email:</label>
+                <InputText id="email" class="form-control" @bind-Value="formData.Email"/>
+            </div>
+        </div>
+        <div class="row">
+            <div class="col-md-12">
+                <label for="password" class="form-label">Password:</label>
+                <InputText type="password" id="password" class="form-control" @bind-Value="formData.Password"/>
+            </div>
+        </div>
+        <div class="row">
+            <div class="col-md-12">
+                <label for="confirm-password" class="form-label">Confirm Password:</label>
+                <InputText type="password" id="confirm-password" class="form-control" @bind-Value="formData.ConfirmPassword"/>
+            </div>
+        </div>
```

Comment:
```suggestion
        <div class="row">
            <div class="col-md-12">
                <label for="name" class="form-label">Name:</label>
                <InputText id="name" class="form-control" @bind-Value="formData.Name"/>
                <ValidationMessage For="() => formData.Name" class="alert alert-danger" role="alert" />
            </div>
        </div>
        <div class="row">
            <div class="col-md-12">
                <label for="email" class="form-label">Email:</label>
                <InputText id="email" class="form-control" @bind-Value="formData.Email"/>
                <ValidationMessage For="() => formData.Email" class="alert alert-danger" role="alert" />
            </div>
        </div>
        <div class="row">
            <div class="col-md-12">
                <label for="password" class="form-label">Password:</label>
                <InputText type="password" id="password" class="form-control" @bind-Value="formData.Password"/>
                <ValidationMessage For="() => formData.Password" class="alert alert-danger" role="alert" />
            </div>
        </div>
        <div class="row">
            <div class="col-md-12">
                <label for="confirm-password" class="form-label">Confirm Password:</label>
                <InputText type="password" id="confirm-password" class="form-control" @bind-Value="formData.ConfirmPassword"/>
                <ValidationMessage For="() => formData.ConfirmPassword" class="alert alert-danger" role="alert" />
            </div>
        </div>
```


This will give better feedback to the user.